test/offsetarray.jl

```diff
@@ -214,11 +214,11 @@ targets2 = ["(1.0, 1.0)",
             "([1.0], [1.0])",
             "([1.0], [1.0])"]
 @testset "printing of OffsetArray with n=$n" for n = 0:4
-    a = OffsetArray(fill(1.,ntuple(d->1,n)), ntuple(identity,n))
-    show(IOContext(io, :limit => true), MIME("text/plain"), a)
-    @test String(take!(io)) == targets1[n+1]
-    show(IOContext(io, :limit => true), MIME("text/plain"), (a,a))
-    @test String(take!(io)) == targets2[n+1]
+    # a = OffsetArray(fill(1.,ntuple(d->1,n)), ntuple(identity,n))
```

Comment:
Did you forget to uncomment these or is it intentional?

Reply:
Oh, yeah, this is unintentional.  The test code needs to be re-jiggered for the new package/module names — these tests break if you just `include` the test file.  The naive things I tried for 60 seconds didn't work so I just gave up for now and temporarily commented them out.